todos/src/app/service/todoService.ts

```diff
@@ -14,4 +16,44 @@ export class todoService {
   async getListByID(id: number): Promise<ListInterface> {
     return (await (await fetch(`${this.url}/${id}`)).json()) ?? {};
   }
+
+  async postCard(list: ListInterface, card: CardInterface) {
+    return ((await fetch(`${this.url}/${list.id}`, {
+      method: 'POST',
+      body: JSON.stringify(card),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      }
+    })));
+  }
+
+  async deleteCard(card: CardInterface): Promise<boolean> {
+    return (await (await fetch(this.url, {
+      method: 'DELETE',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      }
+    })).json());
+  }
+
+  async deleteData(list: ListInterface, card: CardInterface) {
+    fetch(`&{this.url}/${list.id}/${card.id}`, {
```

Comment:
You should do an await here so that the promise gets observed when it succeeds or fails